internal/sirius/client.go

```diff
@@ -14,6 +15,18 @@ func (e ClientError) Error() string {
 	return string(e)
 }
 
+type ValidationErrors map[string]map[string]string
+
+type ValidationError struct {
+	Message string
+	Errors  ValidationErrors
+}
+
+func (ve *ValidationError) Error() string {
```

Comment:
I don't think you need this to be on the pointer receiver (`ValidationError` vs `*ValidationError`), since you never mutate it and it is quite a small struct (string + pointer)

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#receiver-type